plothist/hep_plotters.py

```diff
@@ -121,6 +121,29 @@ def compare_data_mc(
             uncertainties_high**2,
         )
         data_hist[:] = np.stack([data_hist.values(), data_variances], axis=-1)
+    elif comparison_kwargs["comparison"] == "ratio":
+        if comparison_kwargs["ratio_uncertainty"] == "split":
+            np.seterr(divide="ignore", invalid="ignore")
+            # Compute asymmetrical uncertainties to plot_comparison()
+            comparison_kwargs["yerr"] = [
+                uncertainties_low / mc_hist_total.values(),
+                uncertainties_high / mc_hist_total.values(),
+            ]
+            np.seterr(divide="warn", invalid="warn")
+        elif comparison_kwargs["ratio_uncertainty"] == "uncorrelated":
+            data_hist_high = data_hist.copy()
+            data_hist_high[:] = np.stack(
+                [data_hist_high.values(), uncertainties_high**2], axis=-1
+            )
+            data_hist_low = data_hist.copy()
+            data_hist_low[:] = np.stack(
+                [data_hist_low.values(), uncertainties_low**2], axis=-1
+            )
+            # Compute asymmetrical uncertainties to plot_comparison()
+            comparison_kwargs["yerr"] = [
```

Comment:
if user provides an "yerr", this will overwrite it. This comportment might be ok, but needs to be reported in docstring.

Reply:
fixed using `set_default()`